worker/src/core/config.ts

```diff
@@ -334,6 +341,31 @@ const config: Config<ConfigSchema> = convict({
       format: 'required-string',
     },
   },
+  phonebook: {
+    enabled: {
+      doc: 'Kill switch of phonebook related features',
+      default: false,
+      env: 'PHONEBOOK_FEATURE_ENABLE',
+    },
+    endpointUrl: {
+      doc: 'Endpoint url of phonebook server',
+      default: 'http://localhost:8080',
+      env: 'PHONEBOOK_URL',
+      format: 'required-string',
+    },
+    apiKey: {
+      doc: 'API key to make requests to Phonebook',
+      default: 'API_KEY',
+      env: 'PHONEBOOK_API_KEY',
+      format: 'required-string',
+    },
+    version: {
+      doc: 'Version number of Phonebook',
+      default: '1',
```

Comment:
[suggestion] I was thinking we can collapse this env var into the `endpointUrl` (i.e. so instead of `https://phonebook.com/` do `https://phonebook.com/v1`)

Reply:
refactored to use one config variable instead of 2